.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+.idea/
```

Comment:
If you start putting IDE-specific files in the project's `.gitignore`, you'd have to add Eclipse, Netbeans, etc. files too.

Those exclusions should go into a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). Notice how the OS-specific and IDE-specific sample ignore rules are put into a [`Global` subfolder](https://github.com/github/gitignore/tree/master/Global) in https://github.com/github/gitignore/ (and don't miss the [`README.md` file](https://github.com/github/gitignore/tree/master/Global#readme) in that subfolder.